### ISSUE TITLE
[BUG] florists blessing repair placement error

### ISSUE THREAD

**Issue by GamingInfinite** [dpinela/Transcendence#19]
When I was trying to start a rando with Transcendence and it failed to start game because of the following error:
`
[INFO]:[RandomizerMod] - System.MissingMethodException: void ItemChanger.Cost.set_Recurring(bool)
[INFO]:[RandomizerMod] -   at Transcendence.Transcendence.PlaceFloristsBlessingRepair () [0x00001] in /Users/dpinela/dev/hkmod/Transcendence/Transcendence/Transcendence.cs:419 
[INFO]:[RandomizerMod] -   at Transcendence.Transcendence.PlaceItemsRando () [0x0002c] in /Users/dpinela/dev/hkmod/Transcendence/Transcendence/Transcendence.cs:349 
[INFO]:[RandomizerMod] -   at Transcendence.Transcendence.PlaceItems (On.UIManager+orig_StartNewGame orig, UIManager self, System.Boolean permaDeath, System.Boolean bossRush) [0x00024] in /Users/dpinela/dev/hkmod/Transcendence/Transcendence/Transcendence.cs:315 
[INFO]:[RandomizerMod] -   at (wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.Hook<UIManager::StartNewGame>?-1099852800(UIManager,bool,bool)
[INFO]:[RandomizerMod] -   at (wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.Trampoline<UIManager::StartNewGame>?-387037184(UIManager,bool,bool)
[INFO]:[RandomizerMod] -   at MenuChanger.MenuChangerMod.OnStartGame (On.UIManager+orig_StartNewGame orig, UIManager self, System.Boolean permaDeath, System.Boolean bossRush) [0x00005] in C:\dev\MenuChanger\MenuChanger\MenuChangerMod.cs:88 
[INFO]:[RandomizerMod] -   at (wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.Hook<UIManager::StartNewGame>?-846220544(UIManager,bool,bool)
[INFO]:[RandomizerMod] -   at RandomizerMod.Menu.RandomizerMenu.StartRandomizerGame () [0x00010] in C:\dev\RandomizerMod\RandomizerMod\Menu\RandomizerMenu.cs:1085 
`

This is just an issue with PlaceFloristsBlessingRepair and the current version of ItemChanger presumably, because I tested loading a new classic save and it just froze afterwards, presenting this error in the modlog
`
[ERROR]:[UNITY] - MissingMethodException: void ItemChanger.Cost.set_Recurring(bool)
[ERROR]:[UNITY] - Transcendence.Transcendence.PlaceFloristsBlessingRepair () (at /Users/dpinela/dev/hkmod/Transcendence/Transcendence/Transcendence.cs:419)
[ERROR]:[UNITY] - Transcendence.Transcendence.PlaceItems (On.UIManager+orig_StartNewGame orig, UIManager self, System.Boolean permaDeath, System.Boolean bossRush) (at /Users/dpinela/dev/hkmod/Transcendence/Transcendence/Transcendence.cs:322)
[ERROR]:[UNITY] - (wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.Hook<UIManager::StartNewGame>?-600028032(UIManager,bool,bool)
[ERROR]:[UNITY] - (wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition.Trampoline<UIManager::StartNewGame>?32378880(UIManager,bool,bool)
`

Basically the same error, with some extra stuff at the end indicating that it was from the button that was used to say start game but you get the point.

**Comment by GamingInfinite** [comment]
nvm this was just me being stupid.
Reinstalled itemchanger and it works now
